CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.9.2 (2021-04-13)
+
+- Fixed the key info component to use pixel padding instead of the CSS content property. [#161](https://github.com/blackbaud/skyux-indicators/pull/161)
```

Comment:
Is `key info` correct? The changes are actually to the chevron component.

Reply:
Wrong link! Fixed it.